audio/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -52,13 +53,33 @@
         android:layout_height="wrap_content"
         android:includeFontPadding="false"
         android:paddingStart="@dimen/marginTwo"
-        android:paddingRight="@dimen/marginTwo"
+        android:paddingEnd="@dimen/marginTwo"
         android:textColor="@color/audioDurationTextColor"
         android:textSize="@dimen/textFifteen"
         app:layout_constraintStart_toEndOf="@id/divider"
         app:layout_constraintTop_toTopOf="parent"
         tools:text="1:30" />
 
+    <Spinner
```

Comment:
`AppCompatSpinner`?

Reply:
Nah, it'll get handled by https://github.com/aosp-mirror/platform_frameworks_support/blob/d79202da157cdd94c2d0c0b6ee57170a97d12c93/appcompat/src/main/java/androidx/appcompat/app/AppCompatViewInflater.java#L117 - though (as you pointed out offline) a `Switch` wouldn't.